test/e2e/telemeter_test.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2019 The Cluster Monitoring Operator Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/openshift/cluster-monitoring-operator/test/e2e/framework"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+// TestTelemeterRemoteWrite verifies that the monitoring stack can send data to
+// the telemeter server using the native Prometheus remote write endpoint.
+func TestTelemeterRemoteWrite(t *testing.T) {
```

Comment:
Small curiosity questions: 
1. Where does CMO get the config/info that Prom is supposed to remote-write to `https://infogw.api.openshift.com.+`? I see we are enabling remote-write but I don't get how it knows it's supposed to send them there.
2. Why are we modifying the deployment instead of using the ConfigMap

Reply:
I think it's hardcoded in the `SetRemoteWrite` function https://github.com/openshift/cluster-monitoring-operator/blob/master/pkg/manifests/config.go#L449. This is called in `Operator.sync` https://github.com/openshift/cluster-monitoring-operator/blob/master/pkg/operator/operator.go#L608.

Reply:
Indeed, thank you 👍

Reply:
yes, we have no way to turn on native remote write without setting CMO into unmanaged state and hacking the deployment args.